DataProcessing/datax-host/src/main/scala/datax/checkpoint/EventhubCheckpointer.scala

```diff
@@ -57,7 +57,7 @@ object EventhubCheckpointer {
     }
 
     HadoopClient.writeHdfsFile(checkpointFile,
-      offsets.map(v=>v._1+OffsetTokenSeparator+v._2+OffsetTokenSeparator+v._3+OffsetTokenSeparator+v._4+OffsetTokenSeparator+v._5).mkString("\n"), true)
```

Comment:
is behavior the same as before here?

Reply:
i think it is, if you can confirm.

Reply:
yes, adding directWrite=false will remain the same as before